src/components/popup/popup.md

```diff
@@ -0,0 +1,9 @@
+The Popup component can be used to show something under a conditional of any sort:
+
+```js
+<Popup show={false}>This text will not show up</Popup>
+```
+
+```js
+<Popup show>This text will show up</Popup>
+```
```

Comment:
We should give an example that shows off the usage of `children` elements as well.

Reply:
We ended up deleting this and we will explore making an accessible popup later down the road.